apps/nextjs-website/react-components/types/PressRelease/PressRelease.types.ts

```diff
@@ -0,0 +1,9 @@
+export interface PressReleaseProps {
+  title: string;
+  body: JSX.Element;
+  eyelet: string;
+  subtitle?: JSX.Element;
+  ctaText: string;
+  ctaHref: string;
```

Comment:
```suggestion
  link: {
    label: string;
    href: string;
  }
```
Following suggestion in src/app/lib/fetch/types/PageSection.ts